docs/FtpClient.md

```diff
@@ -62,7 +62,8 @@ FtpClient::upload($localFile, $remoteFile, $resume = true, $mode = FTP_BINARY)
 
 ```php
 $interval = 1;
-$client->asyncDownload('illustrations/assets.zip', 'assets.zip', function ($state) use ($interval) {
+$client->setCurrentDir('illustrations');
```

Comment:
Hi, thanks for your PR and for using this lib.
I don't think the solution here is to use `setCurrentDir` because the directory was already specified within the first `asyncDownload` parameter, I've fixed an issue related to `asyncDownload` in #8 and I will soon release a new version of this library.

Reply:
Hi, thank you for your fast response and great library. We've just started to using it on our project. It's good to hear you are working on new version of that. 😃

Reply:
I appreciate your contribution and for your trust in this library,  the version 1.2.4 was just released.